### PR TITLE
Week 20 Updates

### PR DESCRIPTION
**Pull request by Br4nd0R**

### CODE REVIEW

File: .github/workflows/unit-tests.yml [hunk 1]
```diff
@@ -1,86 +1,86 @@
 name: Unit Tests
 
 on:
-  workflow_dispatch:
-  pull_request:
-    types: [opened, synchronize, reopened] # Default
+    workflow_dispatch:
```
Comment (jstvz) [inline .github/workflows/unit-tests.yml:4]: ⬇️ : Why 4 spaces?

Other things that use two spaces:
1. All the Github actions docs
2. Prettier, by default
3. CircleCI docs
4. Ansible docs
5. Learn X in Y
6. This file before this PR (😏 )

**Review by jstvz — DISMISSED** [review]
---
<details>
<summary>Emoji Key</summary>


| Emoji | Meaning                                          | Blocking |
|-------|--------------------------------------------------|----------|
| ±     | Requested change                                 | Yes      |
| ❓    | Query (Request for context or explanation)       | Yes      |
| ❔    | Query (Request for context or explanation)       | No       |
| 💭    | Note/Question to myself or the @-mentioned user. | No       |
| ↔     | "Here's another way to do this:"                 | No       |
| ⬇     | Aesthetic or stylistic quibble                   | No         |

</details>

File: .github/workflows/unit-tests.yml (hunk 1 above)
Comment (Br4nd0R) [reply .github/workflows/unit-tests.yml:4]: Let me revert. It was just what I have my default spacing set to on save.

Comment (jstvz) [reply .github/workflows/unit-tests.yml:4]: Merge it, it's fine.

File: requirements/prod.in [hunk 2]
```diff
@@ -2,7 +2,7 @@ ansi2html
 boto3
 coloredlogs
 cumulusci
-django<3.2
+django<4
```
Comment (jstvz) [inline requirements/prod.in:5]: I spent so much time ranting about indentation that I forgot to mention that a comment explaining the pin would be helpful.

Sorry.

**Review by jstvz — APPROVED** [review]

**Pull request merged**